crates/bevy_math/src/bounding/bounded3d/primitive_impls.rs

```diff
@@ -323,25 +326,26 @@ impl Bounded3d for Triangle3d {
     /// The [`Triangle3d`] implements the minimal bounding sphere calculation. For acute triangles, the circumcenter is used as
     /// the center of the sphere. For the others, the bounding sphere is the minimal sphere
     /// that contains the largest side of the triangle.
-    fn bounding_sphere(&self, translation: Vec3, _rotation: Quat) -> BoundingSphere {
+    fn bounding_sphere(&self, isometry: Isometry3d) -> BoundingSphere {
         if self.is_degenerate() || self.is_obtuse() {
             let (p1, p2) = self.largest_side();
+            let (p1, p2) = (Vec3A::from(p1), Vec3A::from(p2));
             let mid_point = (p1 + p2) / 2.0;
             let radius = mid_point.distance(p1);
-            BoundingSphere::new(mid_point + translation, radius)
+            BoundingSphere::new(mid_point + isometry.translation, radius)
         } else {
             let [a, _, _] = self.vertices;
 
             let circumcenter = self.circumcenter();
             let radius = circumcenter.distance(a);
-            BoundingSphere::new(circumcenter + translation, radius)
+            BoundingSphere::new(circumcenter + Vec3::from(isometry.translation), radius)
```

Comment:
I think this should rather convert `circumcenter`, since `BoundingSphere::new` converts to `Vec3A` anyway.
```suggestion
            BoundingSphere::new(Vec3A::from(circumcenter) + isometry.translation, radius)
```